amaechi-chuks/Politico
# #163500399 Set up and configure postgres

**What does this PR do?**
This PR set up postgres to persist data from the database

**Description of task to be completed**
- Install postgress
- create SQL schema
- establish connection to begin persist data from postgres database

**How should this be manually tested**
- Clone the repo: Run git clone https://github.com/amaechi-chuks/Politico/pull/39 in the folder where you want the repo to be saved.
- start the server using npm run start:dev
- You will see a message on the terminal " database connection established"

**What are the necessary pivotal tracker stories? **
https://www.pivotaltracker.com/story/show/163500399

[Delivers #163500399]
